### ISSUE TITLE
0.34 fails to build

### ISSUE THREAD

**Issue by cromerc** [OpenRC/openrc#170]
I am getting this when building 0.34:
```
supervise-daemon.c:450:23: error: ‘SIGUNUSED’ undeclared (first use in this function); did you mean ‘SI_USER’?
  signal_setup_restart(SIGUNUSED, handle_signal);
                       ^~~~~~~~~
                       SI_USER
supervise-daemon.c:450:23: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [../../mk/prog.mk:35: supervise-daemon.o] Error 1
```

**Comment by williamh** [comment]
I guess you have glibc 2.26. This wasn't caught by me because our stable glibc in Gentoo is older, but it is fixed in 0.34.1.

Thanks,

William

**Comment by cromerc** [comment]
Correct, in both Artix and Cromnix linux we are on glibc 2.26. Will try out 0.34.1 tomorrow.
